docs/pages/reference/workload-identity/sigstore-attestation.mdx

```diff
@@ -0,0 +1,314 @@
+---
+title: Sigstore Workload Attestation
+description: Using Teleport's integration with Sigstore to ensure workload supply chain security
+---
+
+By using Teleport's integration with [Sigstore](https://sigstore.dev), you can
+lock down access to a workload identity and the resources it protects to only
+signed container images, reducing the scope for
+[supply chain attacks](https://www.cloudflare.com/en-gb/learning/security/what-is-a-supply-chain-attack/).
```

Comment:
Perhaps worth linking to a non-Cloudflare resource here if you can find one - you used a NCSC one in the blog post that looks suitable.

Reply:
[e7e55e5](https://github.com/gravitational/teleport/pull/54482/commits/e7e55e551d4e6ddfc48e6a8c451a91b4611da54b)